source/StigData/Processed/Google-Chrome-2.4.xml

```diff
@@ -291,23 +268,23 @@ This policy disables the listed protocol schemes in Google Chrome, URLs using a
       <DuplicateOf />
       <Ensure>Present</Ensure>
       <IsNullOrEmpty>False</IsNullOrEmpty>
-      <Key>HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome\URLBlacklist</Key>
+      <Key>HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome\URLBlocklist</Key>
```

Comment:
This path needs to be validated against the Chrome policy.  Meaning the policy admx should be used to configure this policy and then compare it to what setting is configured in the registry.  I do not believe this rule is correct.  In either case, it does not parse the same as the previous conversion, the valueType is different.

Reply:
Appears to be correct:
![google](https://user-images.githubusercontent.com/20539446/128281611-9165fe6a-9278-4a09-82f4-3e095ab7f0b4.JPG)

Reply:
I added the javascript://* to the GPO
![google2](https://user-images.githubusercontent.com/20539446/128282491-434b723d-ba02-479e-b5d1-58e443d7e7c6.JPG)

Reply:
This rule is incorrect, the registry reflects REG_SZ but the rule parsed as DWORD